CountOnMe/Controller/ViewController.swift

```diff
@@ -8,7 +8,6 @@
 
 import UIKit
 
-
 final class ViewController: UIViewController {
```

Comment:
**suggestion (non-blocking)**: Kudos for the `final` keyword! One suggestion would be to rename the class to something more meaningful like `CalculatorViewController` as `ViewController` means so little by itself.